kong/router.lua

```diff
@@ -770,6 +770,27 @@ local function categorize_route_t(route_t, bit_category, categories)
 end
 
 
+local function sanitize_uri_postfix(uri_postfix)
+  if not uri_postfix or uri_postfix == "" then
+    return uri_postfix
+  end
+
+  if uri_postfix == "." or uri_postfix == ".." then
+    return ""
+  end
+
+  if sub(uri_postfix, 1, 2) == "./" then
+    return sub(uri_postfix, 3)
```

Comment:
@dndx,

do you think this should be:
```
    return sub(uri_postfix, 3)
```

or 

```
    return sub(uri_postfix, 2)
```

Reply:
I think `return sub(uri_postfix, 3)` is correct, if there isn't a `/` at the beginning, we should not add another one either. Although from the path handling code it seems to also detect for double slashes, but better to keep it the same. The current code looks fine to me.

Reply:
@dndx, it mostly affects the `v1` path handling.

Reply:
@bungle Will it break `v1` path handling?